packages/web3-utils/src/index.js

```diff
@@ -41,7 +42,7 @@ var randombytes = require('randombytes');
  * @param {any} optionalData
  * @return {Object} the emitter
  */
-var _fireError = function (error, emitter, reject, callback, optionalData) {
+export var _fireError = function (error, emitter, reject, callback, optionalData) {
```

Comment:
Are we sure than all tree shaking systems can handle this? I'm not sure if removing `var` references can be considered safe, as they can be mutated from other modules.

Anyway, what's to win by using `export var asd = function()`? `export function asd()` is shorter and WAY more clear.

Reply:
PS: @nivida please don't take this message as something personal, I know this comes from a time before you joined the project :)

Reply:
I just kept anything as it is and added the bare ES imports/exports without doing any further changes. But yes such stuff could get improved but doesn't really make a difference.

Reply:
I tested this against webpack in production mode, and in dev mode `optimization.usedExports`. In the first case, unused utils functions are actually removed. In the second one they are flagged for removal.

This doesn't mean that they get tree-shaken everywhere, but I think it's enough for making this as resolved.